lbz/events/broker.py

```diff
@@ -8,17 +8,21 @@
 
 logger = get_logger(__name__)
 
+EVENT_BRIDGE_DEFAULT_TYPE_KEY = "detail-type"
+EVENT_BRIDGE_DEFAULT_DATA_KEY = "detail"
+COGNITO_DEFAULT_TYPE_KEY = "triggerSource"
+COGNITO_DEFAULT_DATA_KEY = "request"
```

Comment:
I don't think we should create global variables for these strings, it gives the ability to manipulate them, but also they will only be used once:
```suggestion
```